contracts/linkStaking/CommunityVaultAutomation.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.15;
+
+import {CommunityVCS} from "./CommunityVCS.sol";
+import {IVault} from "./interfaces/IVault.sol";
+import {AutomationCompatibleInterface} from "@chainlink/contracts/src/v0.8/interfaces/AutomationCompatibleInterface.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+contract CommunityVaultAutomation is AutomationCompatibleInterface, Ownable {
+    CommunityVCS internal communityVCS;
+    uint256 public minRewardsTotal;
+    uint256 public minRewardsPerVault;
+
+    constructor(
+        address _communityVCS,
+        uint256 _minRewardsTotal,
+        uint256 _minRewardsPerVault
+    ) {
+        communityVCS = CommunityVCS(_communityVCS);
+        minRewardsTotal = _minRewardsTotal;
+        minRewardsPerVault = _minRewardsPerVault;
+    }
+
+    /**
+     * @notice returns whether or not rewards is equal to or greater than the minimum rewards set
+     * @return upkeepNeeded whether or not rewards should be claimed
+     * @return performData abi encoded total claimable rewards and list of vault indexes to claim from
+     *
+     */
+    function checkUpkeep(bytes calldata) external returns (bool upkeepNeeded, bytes memory performData) {
+        (uint256 _totalRewards, uint256[] memory _vaults) = checkRewards();
```

Comment:
Not saying one way is better than the other but for the sake of consistency, I try to avoid using underscores for any variables except function params which I always use them for

Reply:
Makes sense. I usually used them for anything that wasnt global to discern between the two but ill go through and update them.